setup.py

```diff
@@ -283,7 +283,7 @@ def do_setup():
             'flask-swagger==0.2.13',
             'flask-wtf>=0.14.2, <0.15',
             'funcsigs==1.0.0',
-            'future>=0.16.0, <0.17',
+            'future>=0.17.0, <0.18',
```

Comment:
Can we still leave 'future>=0.16.0 intact?

Reply:
yep